tests/test_downsampling.py

```diff
@@ -1,204 +1,204 @@
-    )
-
-
-def test_downsample_cube_job():
-    downsample_test_helper(False)
-
-
-def test_compressed_downsample_cube_job():
-    downsample_test_helper(True)
-
-
-def test_downsample_multi_channel():
-    source_info = WkwDatasetInfo("testoutput/multi-channel-test", "color", "uint8", 1)
-    target_info = WkwDatasetInfo("testoutput/multi-channel-test", "color", "uint8", 2)
-    try:
-        shutil.rmtree(source_info.dataset_path)
-        shutil.rmtree(target_info.dataset_path)
-    except:
-        pass
-
-    offset = (0, 0, 0)
-    num_channels = 3
-    size = (32, 32, 10)
-    source_data = (
-        128 * np.random.randn(num_channels, size[0], size[1], size[2])
-    ).astype("uint8")
-    file_len = 32
-
-    with open_wkw(
-        source_info, num_channels=num_channels, file_len=file_len
-    ) as wkw_dataset:
-        print("writing source_data shape", source_data.shape)
-        wkw_dataset.write(offset, source_data)
-    assert np.any(source_data != 0)
-
-    downsample_cube_job(
-        source_info,
-        target_info,
-        (2, 2, 2),
-        InterpolationModes.MAX,
-        CUBE_EDGE_LEN,
-        tuple(a * WKW_CUBE_SIZE for a in offset),
-        False,
-    )
-
-    channels = []
-    for channel_index in range(num_channels):
-        channels.append(
-            downsample_cube(
-                source_data[channel_index], (2, 2, 2), InterpolationModes.MAX
-            )
-        )
-    joined_buffer = np.stack(channels)
-
-    target_buffer = read_wkw(
-        target_info,
-        tuple(a * WKW_CUBE_SIZE for a in offset),
-        list(map(lambda x: x // 2, size)),
-        file_len=file_len,
-    )
-    assert np.any(target_buffer != 0)
-
-    assert np.all(target_buffer == joined_buffer)
-
-def test_anisotropic_mag_calculation():
-    mag_tests = [
-        [(10.5, 10.5, 24),      Mag(1),         Mag((2, 2, 1))],
-        [(10.5, 10.5, 21),      Mag(1),         Mag((2, 2, 1))],
-        [(10.5, 24, 10.5),      Mag(1),         Mag((2, 1, 2))],
-        [(24, 10.5, 10.5),      Mag(1),         Mag((1, 2, 2))],
-        [(10.5, 10.5, 10.5),    Mag(1),         Mag((2, 2, 2))],
-        [(10.5, 10.5, 24),      Mag((2, 2, 1)), Mag((4, 4, 1))],
-        [(10.5, 10.5, 21),      Mag((2, 2, 1)), Mag((4, 4, 2))],
-        [(10.5, 24, 10.5),      Mag((2, 1, 2)), Mag((4, 1, 4))],
-        [(24, 10.5, 10.5),      Mag((1, 2, 2)), Mag((1, 4, 4))],
-        [(10.5, 10.5, 10.5),    Mag(2),         Mag(4)],
-    ]
-    for i in range(len(mag_tests)):
-        assert mag_tests[i][2] == get_next_anisotropic_mag(mag_tests[i][1], mag_tests[i][0]), "The next anisotropic" \
-                                                                                          " Magnification of {} with " \
-                                                                                          "the size {} should be {}"\
-            .format(mag_tests[i][1], mag_tests[i][0], mag_tests[i][2])
+# import logging
```

Comment:
Comments?

Reply:
commented the test out since it takes to much time.